environments/.caas/inventory/group_vars/all/cluster.yml

```diff
@@ -20,3 +20,7 @@ openondemand_servername_default: "{{ hostvars[groups['openstack'][0]].cluster_ga
 openondemand_servername: "{{ zenith_fqdn_ood | default(openondemand_servername_default) }}"
 
 appliances_state_dir: /var/lib/state
+
+# Defaults for caas-provided extravars:
+cluster_home_manila_share: false
+cluster_project_manila_share: false
```

Comment:
would this work?
```
cluster_project_manila_share: false
cluster_home_manila_share: "{{ cluster_project_manila_share }}"

Reply:
It could, but why? Are you trying to only set one extravar from the cluster template or something?